lib/rssbridge.php

```diff
@@ -51,8 +51,7 @@
 /** URL to the RSS-Bridge repository */
 define('REPOSITORY', 'https://github.com/RSS-Bridge/rss-bridge/');
 
-// Functions
-require_once PATH_LIB . 'Exceptions.php';
+// Files
```

Comment:
What does this comment mean? “Functions” explained why the files cannot be autoloaded (though it would be nice to have the comment say it explicitly) but I have no idea what information does “Files” convey.

Reply:
These are files that contain more than just functions. There are global consts and exception classes in them too. `Files` means that they must always be included with e.g. composer's `files` autoload option.